timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosResourcesFactory.java

```diff
@@ -150,10 +150,18 @@ private static PaxosResources configureLeaderForAllClients(
 
```

Comment:
the above network client factory builder and leader pinger factory builder should use the `PinningX`, not the `SingleLeaderX`.